package.json

```diff
@@ -5,9 +5,11 @@
   "main": "index.js",
   "scripts": {
     "build": "razzle build",
+    "build-storybook": "build-storybook -s public -o .out",
```

Comment:
This line is only needed for building Storybook locally. If you only want to build it for deployment to `latest` on CI, you can remove this line, since the CI step used in the .travis.yml ignores this.